torch/_inductor/config.py

```diff
@@ -361,6 +361,21 @@ def decide_compile_threads():
 # Pad input tensors of matmul/bmm/addmm to leverage Tensor Cores in NVIDIA GPUs
 shape_padding = os.environ.get("TORCHINDUCTOR_SHAPE_PADDING", "1") == "1"
 
+# Whether to shape-pad only along K dimension
+# which does not affect output shape of GEMM
+shape_pad_only_k_dim: bool = False
+
+# When, during shape padding, dimension N would have to be padded, but
+# dimension M would not, then we can avoid a padding if we perform an
+# explicit transpose ( e.g. matmul(A,B) = matmul(B.T, A.T) ) in order to
```

Comment:
This should be
```
matmul(A,B) = matmul(B.T, A.T).T
```
instead
?

Reply:
right, good it's just in the comment ;) I'll fix it..